P1_BMI_Stats.R

```diff
@@ -32,31 +32,35 @@ library(here)
 library(readr)
```

Comment:
Delete all of the `library()` calls and instead add them to the `setup_environment.R` script.